remote-api/src/main/java/org/alfresco/rest/api/nodes/NodeRuleSetsRelation.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.rest.api.nodes;
+
+import org.alfresco.rest.framework.resource.RelationshipResource;
+import org.alfresco.service.Experimental;
+
+/**
+ * Folder node rule sets.
+ *
+ */
+@Experimental
+@RelationshipResource(name = "rulesets", entityResource = NodesEntityResource.class, title = "Folder node rule sets")
+public class NodeRuleSetsRelation
```

Comment:
Ugh! I thought we used the term "ruleset", but Googling suggests we call it a "rule set."  I think we should change the URL too.

Reply:
I'll fix it.